src/button.tsx

```diff
@@ -81,6 +82,22 @@ const renderButtonElement = ({
 };
 
 const ButtonLinkChevron: React.FC = () => {
+    const {skinName} = useTheme();
+
+    // vivo new skin has a different chevron
+    if (skinName === VIVO_NEW_SKIN) {
+        return (
+            // w = 4px / 14px = 0.2857em
+            // h = 6px / 14px = 0.4286em
+            <svg width="0.2857em" height="0.4286em" viewBox="0 0 4 6" fill="none">
```

Comment:
please, review this sizes. I copied the svg from figma with a size of 4x6px and translated it to `em` (button links have a font size of 14)

Reply:
I think you didn't copy the correct layer

The container should be 0.5em like in the line 102
![image](https://github.com/Telefonica/mistica-web/assets/6722153/16714afa-1fdb-4cbc-ad10-00915b929ee2)

Reply:
I guess that's why the chevron looks weird:
![image](https://github.com/Telefonica/mistica-web/assets/579417/e52fb68e-db35-4949-a813-4cca04e05d8b)

Reply:
copying from figma is a nightmare :(

Reply:
should be fixed now